code/sql/algoliasearch_setup/mysql4-upgrade-1.5.0-1.6.0RC.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
```

Comment:
> We need to be careful with the filename, because I am not sure what happens when you release 1.6.0

With releasing 1.6.0 I want to rename the file. There might be an issue that 2 same files might be there - RC and non-RC. Question is how big issue is that or what other options we have?

> In fact I would rather empty the queue

I was thinking about that but if you empty the queue completly you would loose operations. On the other hand, you must force complete re-index because of the BC breaks. Yes. I will update it to deleting all rows from queue.